.travis.yml

```diff
@@ -1,16 +1,17 @@
-language: node_js
-node_js:
-- '4'
+sudo: true
+
+language: cpp
+
+services:
+  - docker
+
+before_install:
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/install-package.sh
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/build-package.sh
+  - export PACKAGE=${TRAVIS_REPO_SLUG#*/}
+
 install:
-- npm install
```

Comment:
I'm pretty sure this is in here for a reason. Is there a benefit from docker here even?